openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/DatabaseSchemaPage.component.tsx

```diff
@@ -147,7 +147,7 @@ const DatabaseSchemaPage: FunctionComponent = () => {
       position: 1,
     },
     {
-      name: 'Activity Feed',
+      name: 'Activity Feed & Task',
```

Comment:
Can we keep this as `Activity Feed`, as the task is not supported for the database and database schema entity?